### PR TITLE
Torpedo deposit prefab

### PR DESCRIPTION
**Pull request by MatheusIsNotAvailable**
[FEATURE]

## About the PR 
Adds a new water prefab, a torpedo deposit underseas, it's keeper has been long dead, it contains 4 random torpedoes and a pretty cool backstory told trough the environment.
## Why's this needed? 
Torpedoes are fun to use, it Is a good place for antags to go and then bring them on station, will allow people to use them on Oshan too. 

## Changelog

```changelog
(u)Arthur Holiday 
(+)Added a new prefab to water maps, a torpedo deposit, it contains a few torpedoes and a cool back story. Check it out!
```

### CODE REVIEW

**Comment by DimWhat** [comment]
Would prefer that something with incredibly deadly stuff would have something defending it. Lore or not.

**Comment by MatheusIsNotAvailable** [comment]
What could I add to it? Maybe some sea mines around it?

**Comment by alexeric123** [comment]
> What could I add to it? Maybe some sea mines around it?

Maybe some gunbots/sharkdrones?

**Comment by DimWhat** [comment]
Shark drones, fish drones, normal drones, some mean critters or something like that. The place is abandoned and it probably means it's open for something else to move in.

**Review by flrsh — APPROVED** [review]
More prefabs good.

File: code/WorkInProgress/torpedoes.dm [hunk 1]
```diff
@@ -550,6 +550,16 @@
 		changeIcon()
 		return
 
+/obj/torpedo_tray/random_loaded
+	icon = 'icons/obj/32x64.dmi'
+	icon_state = "emptymissiletray"
+	New()
+		..()
+		var/obj/torpedo/hiexplosive/T = pick(new/obj/torpedo/toxic,new/obj/torpedo/incendiary,new/obj/torpedo/hiexplosive,new/obj/torpedo/explosive)
```
Comment (ZeWaka) [inline code/WorkInProgress/torpedoes.dm:558]:
```suggestion
		var/obj/torpedo/T = pick(new/obj/torpedo/toxic,new/obj/torpedo/incendiary,new/obj/torpedo/hiexplosive,new/obj/torpedo/explosive)
```
nope

File: code/WorkInProgress/torpedoes.dm [hunk 2]
```diff
@@ -550,6 +550,16 @@
...
+		src.loaded = T
+		T.set_loc(src)
+		changeIcon()
+		return
```
Comment (ZeWaka) [inline code/WorkInProgress/torpedoes.dm:562]: useless
```suggestion
```

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**